policyengine_us/variables/gov/states/nj/tax/income/property_tax/nj_property_tax_credit_eligible.py

```diff
@@ -0,0 +1,41 @@
+from policyengine_us.model_api import *
+
+
+class nj_property_tax_credit_eligible(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "New Jersey property tax credit eligibility"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.NJ
+
+    def formula(tax_unit, period, parameters):
+        # Same as deduction eligibility, but also eligible if 65+ or blind/disabled and paid property taxes.
+        deduction_eligibility = tax_unit(
+            "nj_property_tax_deduction_eligible", period
+        )
+
+        # If filing jointly, only one spouse needs to be 65+ or blind/disabled.
+        blind_head = tax_unit("blind_head", period)
+        disabled_head = tax_unit("disabled_head", period)
+        blind_spouse = tax_unit("blind_spouse", period)
+        disabled_spouse = tax_unit("disabled_spouse", period)
+        senior_head = tax_unit("age_head", period) > 65
+        senior_spouse = tax_unit("age_spouse", period) > 65
+        senior_blind_disabled = (
+            blind_head
+            + disabled_head
+            + blind_spouse
+            + disabled_spouse
+            + senior_head
+            + senior_spouse
+        ) > 0
+
+        # Next check if they paid property taxes (either directly or through rent).
+        direct_property_taxes = tax_unit("nj_homeowners_property_tax", period)
```

Comment:
Why use `nj_homeowners_property_tax` over `real_estate_taxes`?

Reply:
Thought the rules around eligible NJ property taxes (only on primary home, restrictions on rooms/bathrooms, etc.) might be more restrictive, but if it's safe to assume they're the same I can remove the NJ variable.

Reply:
Just took a closer look. I think a separate variable makes sense if it's limited to the property taxes that would count for this credit (not just all homeowner property taxes), and if you add `adds = ["real_estate_taxes"]` so that flows through by default.